assets/sidecar/backstage-cr-patches.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: rhdh.redhat.com/v1alpha3
+kind: Backstage
+metadata:
+  name: ai-rh-developer-hub
+  namespace: ai-rhdh
+spec:
+  application:
```

Comment:
Do we need to reference the app config you created below?

Reply:
@johnmcollier and @thepetk and I talked and I now understand that @johnmcollier thinks it better to have a full of the backstage CR in case a developer does not use the installer 

Ultimately that makes our dependencies explicit and clear

I will make the update as part of my next push